src/dmd/backend/outbuf.h

```diff
@@ -57,14 +61,34 @@ struct Outbuffer
     void enlarge(unsigned nbytes);
 
     // Write n zeros; return pointer to start of zeros
-    void *writezeros(d_size_t n);
+    void *writezeros(uint n)
```

Comment:
Why did that code end up in the header ? Wouldn't it be better (and closer to the original) to just move it to the D side instead of duplicating it here ?

Reply:
Inlining is important for speed in modules not converted to D yet.